confluence-xml/checkstyle-suppressions.xml

```diff
@@ -7,6 +7,8 @@
 <suppressions>
   <suppress checks="ClassFanOutComplexity|CyclomaticComplexity|FileLength|JavaNCSS"
             files="src/main/java/org/xwiki/contrib/confluence/filter/input/ConfluenceXMLPackage\.java$"/>
+  <suppress checks="ClassFanOutComplexity|CyclomaticComplexity|FileLength|JavaNCSS"
+    files="src/main/java/org/xwiki/contrib/confluence/filter/internal/input/ConfluenceInputFilterStream\.java$"/>
```

Comment:
You should add JavaNCSS to the line already there for ConfluenceInputFilterStream (next one)